src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/memories/JqlMemory.kt

```diff
@@ -1,7 +1,14 @@
 package com.atlassian.performance.tools.jiraactions.api.memories
 
 import com.atlassian.performance.tools.jiraactions.api.page.IssuePage
+import java.util.function.Predicate
 
 interface JqlMemory : Memory<String> {
+
     fun observe(issuePage: IssuePage)
+
+    @JvmDefault
+    fun recallByTag(filter: Predicate<String>): String? {
+        return null;
```

Comment:
the semicolon at the of the `return null`

Reply:
I'm also concerned about returning `null` as a default implementation. It will make the change source compatible, but I see no value besides that. It'll probably hide the problem until it appears in a different place (no results for the action at the end of the test).

Reply:
Maybe this method should be documented e.g which strings are allowed? Can we somehow validate them? Should we change the name of the method into recallByPredicate? We can throw an Exception as the default implementation.

Reply:
Default recall also returns null when there is no result i think it is better than exception.
In my opinion name is ok because it provides more context. Everybody see that predicate is there